tests/test_docs_validation.py

```
@@ -0,0 +1,23 @@
+Tests cover document structure validation, link checking, configuration validation,
+file format support, error handling, and performance considerations.
+"""
+
+import pytest
+
+# Use pytest.importorskip to conditionally import the real module
+docs_validation = pytest.importorskip("agor.docs_validation")
+
+# Import the functions and classes directly from the module
+from agor.docs_validation import (
+    validate_documentation,
```

Ruff (0.11.9)

`agor.docs_validation.validate_documentation` imported but unused

Remove unused import

(F401)